src/seedu/addressbook/data/person/Name.java

```diff
@@ -37,10 +38,15 @@ public static boolean isValidName(String test) {
     }
 
     /**
-     * Retrieves a listing of every word in the name, in order.
+     * Retrieves a listing of every word in the name, in lower case, in order.
      */
     public List<String> getWordsInName() {
-        return Arrays.asList(fullName.split("\\s+"));
+        List<String> wordsInName = Arrays.asList(fullName.split("\\s+"));
```

Comment:
\\s+ should be stored in a variable with a meaningful name and then used in the code.